commons/src/main/java/br/com/studae/commons/command/adapter/impl/BooleanAdapter.java

```diff
@@ -0,0 +1,33 @@
+package br.com.studae.commons.command.adapter.impl;
+
+import br.com.studae.commons.command.adapter.Adapter;
+import br.com.studae.commons.command.context.Argument;
+import br.com.studae.commons.command.context.Context;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+
+public class BooleanAdapter implements Adapter<Boolean> {
+
+    @Override
+    public Boolean adapt(Context context, Argument argument) {
+        return context.getOptionMapping(argument.getName())
+          .map(OptionMapping::getAsBoolean)
+          .orElse(null);
```

Comment:
Por que não em vez de retornar `null` retornar `false` por padrão?

Reply:
Caso optemos por retornar Optional em vez de valores diretos, isso poderia ficar assim:
```suggestion
          .orElse(Optional.empty());
```